### ISSUE TITLE
Python support for DiffEqGPU

### ISSUE THREAD

**Issue by sibyjackgrove** [SciML/DiffEqGPU.jl#273]
Is there any plan to add Python support for `DiffEqGPU.jl`similar to how `DifferentialEquations.jl` can be used from Python using `diffeqpy`?

**Comment by utkarsh530** [comment]
I haven't tried it yet. Maybe post a MWE?
Something like this should work: https://scicomp.stackexchange.com/questions/36994/cuda-python-for-numerical-integration-and-solving-differential-equations

**Comment by ChrisRackauckas** [comment]
I think they mean like what we have with R. https://www.stochasticlifestyle.com/gpu-accelerated-ode-solving-in-r-with-julia-the-language-of-libraries/ . The key thing we are missing there is that MTK tracing doesn't work in Python.

**Comment by sibyjackgrove** [comment]
> I think they mean like what we have with R. https://www.stochasticlifestyle.com/gpu-accelerated-ode-solving-in-r-with-julia-the-language-of-libraries/ . The key thing we are missing there is that MTK tracing doesn't work in Python.

Yes, this is what I was referring to. It's unfortunate that it is not supported in Python.

**Comment by ChrisRackauckas** [comment]
It has to do with the wrapper library, not necessarily the GPU library.  https://github.com/SciML/diffeqpy/issues/78 and https://github.com/SciML/diffeqpy/issues/57
